src/auth/index.js

```diff
@@ -20,8 +21,33 @@ export default class Auth {
     this.client = client;
     this.rootUrl = rootUrl;
     this.codec = options.codec;
-    this.storage = createStorage(options.storageType);
-    this.providers = createProviders(this);
+    this.platform = options.platform !== undefined ? options.platform : _platform;
```

Comment:
nit: could use `||` instead of ternary here, since i think we probably want the same behavior with other falsy values too